pkg/pillar/kubeapi/vitoapiserver.go

```diff
@@ -271,6 +279,17 @@ func RolloutDiskToPVC(ctx context.Context, log *base.LogObject, exists bool,
 		volSize = actualVolSize
 	}
 
+	// Create PVC and then copy data. We create PVC to set the designated node id label.
+	if !exists {
```

Comment:
do you need to change the args to have 'args = append(args, "--no-create")' after this is now created?

Reply:
It's already there right ? line 312

Reply:
Ok

Reply:
But Ian’s this new code changes the existing meaning, after your crest, it should be Exist now?